query-planner-js/src/buildPlan.ts

```diff
@@ -2495,6 +2500,26 @@ class FetchDependencyGraph {
     };
   }
 
+  private updateUnhandledOnProcessedGroups(
+    processed: readonly FetchGroup[],
+    currentUnhandled: UnhandledGroups,
+    readyToHandle: FetchGroup[],
+  ): UnhandledGroups {
+    const unhandled: UnhandledGroups = [];
+    for (const current of currentUnhandled) {
```

Comment:
minor nit: To be consistent with `mergeRemainings()`/make code more clear, could you change `current` to e.g. `[g, edges]`? (`edges` is more clear than `current[1]`)